src/app/index.ts

```diff
@@ -425,10 +426,12 @@ module.exports = class extends yo {
     this.log(`NOTE: ${chalk.bgGreen('Arguments')} must be specified in the order below, and ${chalk.bgMagenta('Options')} must follow ${chalk.bgGreen('Arguments')}.\n`);
     this.log(`  ${chalk.bgGreen('projectType')}:Specifies the type of project to create. Valid project types include:`);
     this.log(`    ${chalk.yellow('angular:')}  Creates an Office add-in using Angular framework.`);
-    this.log(`    ${chalk.yellow('excel-functions:')} Creates an Office add-in for Excel custom functions.  Must specify 'Excel' as host parameter.`);
+    this.log(`    ${chalk.yellow('excel-functions-shared:')} Creates an Office add-in for Excel custom functions using a Shared Runtime.  Must specify 'Excel' as host parameter.`);
```

Comment:
Don't we automatically set the host parameter?  We wouldn't need this part of the message.

Reply:
I guess it's because the messages are a bit different between themselves. We would need to add a new parameter to all hosts (like `displayMessage`) if we would like to keep current messages.